content/contributing-terms.en.md

```diff
@@ -3,539 +3,29 @@ title: "Contributing terms"
 weight: 2
 ---
 
-# Contributing terms
+# Contributing new declarations
 
-## Tracking new terms
+This is a step by step guide to help you add declarations to the [Contrib collection](https://github.com/OpenTermsArchive/contrib-declarations). This collection is dedicated to volunteer contribution of declarations to Open Terms Archive.
 
-Tracking terms is done by _declaring_ them and the service they are associated with. Such a declaration is achieved by editing JSON files in the [`declarations`](https://github.com/OpenTermsArchive/contrib-declarations/tree/main/declarations) folder.
+Having understood briefly how a declaration is structured in JSON format, we need to look at concrete steps on how you can add these JSON files to the repository.
```

Comment:
At this stage, the reader has not really “understood briefly how a declaration is structured in JSON format”.

Reply:
Made the change